app/controllers/passengers_controller.rb

```diff
@@ -42,6 +42,12 @@ def index
     respond_to do |format|
       format.html
       format.pdf { passenger_pdf }
+      format.csv do
```

Comment:
This seems like the right approach — use the selected filter on the index — but the link you added doesn't use this.